notebooks/MAST/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.ipynb

```diff
@@ -635,8 +644,8 @@
     "\"\"\".format(ra,dec,radius)\n",
```

Comment:
Comment for line 625/634:
`Mast._resolve_object(objname)` -> `Mast.resolve_object(objname)`